### DISCUSSION TITLE
What should be the minimum requirements to metatags or parameters to share with linkedin.?

### DISCUSSION THREAD

**Discussion by renatoalvarez90** [MurhafSousli/ngx-sharebuttons#698] [General]
What should be the minimum requirements to metatags or parameters to share with linkedin.?

**Comment by MurhafSousli** [comment]
To use LinkedIn button, you can only use the URL input, if URL input left empty it will use the current page URL. see https://github.com/MurhafSousli/ngx-sharebuttons/wiki/Meta-tags-inputs-compatibility

Regarding the meta tags, they are unified for most social media. I think LinkedIn uses the `og:image` and `og:title`

After publishing your site, you can test it out via LinkedIn Inspector https://www.linkedin.com/post-inspector/
